{
  "id" : "4b580b9f-9830-4e61-bf3f-d52feb9e21cc",
  "name" : "api_2.0_sql_statements",
  "request" : {
    "url" : "/api/2.0/sql/statements/",
    "method" : "POST",
    "bodyPatterns" : [ {
      "equalToJson" : "{\"statement\":\"INSER INTO main.jdbc_test_schema.query_syntax_error_test_table (id, col1, col2) VALUES (1, 'value1', 'value2')\",\"warehouse_id\":\"dd43ee29fedd958d\",\"session_id\":\"01effa65-683f-1e4f-ae23-14b319b6494b\",\"disposition\":\"EXTERNAL_LINKS\",\"format\":\"ARROW_STREAM\",\"on_wait_timeout\":\"CONTINUE\",\"wait_timeout\":\"10s\",\"parameters\":[],\"result_compression\":\"LZ4_FRAME\"}",
      "ignoreArrayOrder" : true,
      "ignoreExtraElements" : true
    } ]
  },
  "response" : {
    "status" : 200,
    "body" : "{\"statement_id\":\"01effa65-6916-1a00-be04-7ddc75857ec0\",\"status\":{\"state\":\"FAILED\",\"error\":{\"error_code\":\"BAD_REQUEST\",\"message\":\"\\n[PARSE_SYNTAX_ERROR] Syntax error at or near 'INSER'. SQLSTATE: 42601 (line 1, pos 0)\\n\\n== SQL ==\\nINSER INTO main.jdbc_test_schema.query_syntax_error_test_table (id, col1, col2) VALUES (1, 'value1', 'value2')\\n^^^\\n\"},\"sql_state\":\"42601\"}}",
    "headers" : {
      "x-request-id" : "ef93ba6b-022a-4f65-8926-5263f0dbab9f",
      "date" : "Thu, 6 Mar 2025 08:31:33 GMT",
      "server" : "databricks",
      "x-databricks-popp-response-code-details" : "via_upstream",
      "x-databricks-shard-debug" : "oregon-staging",
      "vary" : "Accept-Encoding",
      "x-databricks-org-id" : "6051921418418893",
      "strict-transport-security" : "max-age=31536000; includeSubDomains; preload",
      "x-content-type-options" : "nosniff",
      "x-databricks-popp-routing-reason" : "deployment-name",
      "content-type" : "application/json",
      "x-databricks-apiproxy-response-code-details" : "via_upstream"
    }
  },
  "uuid" : "4b580b9f-9830-4e61-bf3f-d52feb9e21cc",
  "insertionIndex" : 22
}